test/TestConfig/scripts/tools/getDependencies.pl

```diff
@@ -136,19 +136,21 @@
 			}
 		}
 
-		# TODO check samtools.jar file
+		# TODO check asmtools.jar file
 		if (index($jars_info[$i]{fname}, "asmtools") != -1) {
-			my $untar = `tar -zxvf $path/asmtools-6.0.tar.gz -C $path`;
-			if ($? != 0 ) {
-				print $untar;
-				die "ERROR: untar $path/asmtools-6.0.tar.gz failed"
+			if (! -e "$path/asmtools.jar"){
+				my $untar = `tar -zxvf $path/asmtools-6.0.tar.gz -C $path`;
+				if ($? != 0 ) {
+					print $untar;
+					die "ERROR: untar $path/asmtools-6.0.tar.gz failed"
+				}
+				my $unzip = `unzip -o $path/asmtools-6.0.zip -d $path`;
+				if ($? != 0 ) {
+					print $unzip;
+					die "ERROR: untar $path/asmtools-6.0.tar.gz failed"
```

Comment:
You can dong unzip, but message is untar. Instead of writing detailed message, please use variable. Otherwise, it is very easy to become out of sync.

Reply:
updated